### ISSUE TITLE
Replace tmpro inlined sprites with inlined icons

### ISSUE THREAD

**Issue by metachronism** [StrangeLoopGames/EcoIssues#2784]
We need to figure a way to place icons in text without using TMPro's inlined sprites; mainly because our Icons almost always are more than one image in layers; and also modded items arent support at all in this system.

**Comment by MarjieAVolk** [comment]
Mod items aren't supported?  They are as far as I know - it's looking up the item icon prefab in ModBundleManager and then translating it basically, so if mod items don't work now, they won't once this is changed either.

**Comment by metachronism** [comment]
mod items as in items added through the modkit with their own sprites (like greenleafs currency icons etc)

**Comment by MarjieAVolk** [comment]
Yeah, I mean I don't really know how the mod system works, but doesn't ModBundleManager handle mod item icons?

**Comment by Kodekc** [comment]
Need to fix the icon that is used in the description and in the chat
![1](https://user-images.githubusercontent.com/16691965/41862241-fc2777e2-78ab-11e8-8f0a-9f28e3589665.PNG)
![2](https://user-images.githubusercontent.com/16691965/41862242-fc56274a-78ab-11e8-974b-7ef2e3f885c7.PNG)

hash/commit ModKit: [0749fba](https://github.com/StrangeLoopGames/EcoModKit/commit/8d4be15f3a78f670d75ba0780831294f8b0d00c4)

UP feature!

**Comment by MarjieAVolk** [comment]
That's a mod item

**Comment by Kodekc** [comment]
Yes, I ask you to let the modding items set the chat icons

**Comment by MarjieAVolk** [comment]
Isn't that already supported?

**Comment by Kodekc** [comment]
No, I could not add this via ModKit, or there is no documentation of this. (Background and Foreground are filled)
![default](https://user-images.githubusercontent.com/16691965/42654459-5ab8e7e6-8621-11e8-88ab-f5638bab8eb4.PNG)

**Comment by MarjieAVolk** [comment]
Okay, I get what's going on now.  Yeah that's a problem.
